.gitignore

```diff
@@ -1,7 +1,13 @@
+new-file-that-ignored
```

Comment:
This is the comment in second review for new line that added after first review.